### DISCUSSION TITLE
Dependabot stuck at 10h+

### DISCUSSION THREAD

**Discussion by Mailootje** [community/community#147199] [Actions]
![Screenshot_20241216_090647_Chrome.jpg](https://github.com/user-attachments/assets/8e8fd10e-7a1b-40ba-877f-814d9b5d942d)

How can I fix this? My package.json doesnt have over 150 packages

If you need more info let me know!

**Comment by zheli** [comment]
We also have a dependabot action stuck in limbo: https://github.com/Bits-Heaven/bitsheaven-landing-page/actions/runs/13396439445

can't cancel, can't create new ones.
